CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ This file contains notable changes (e.g. breaking changes, major changes, etc.)
 
 This file was introduced starting Kani 0.23.0, so it only contains changes from version 0.23.0 onwards.
 
+## [0.31.0]
+
+## What's Changed
+* Add run_command benchcomp visualization by @karkhaz in https://github.com/model-checking/kani/pull/2542
+* Add `--exact` flag by @jaisnan in https://github.com/model-checking/kani/pull/2527
+* Build the verification libraries using Kani compiler by @celinval in https://github.com/model-checking/kani/pull/2534
+* Verify all Kani attributes in all crate items upfront by @celinval in https://github.com/model-checking/kani/pull/2536
+* Install doc dependencies before building by @karkhaz in https://github.com/model-checking/kani/pull/2564
+* Bump CBMC version to 5.86.0 by @zhassan-aws in https://github.com/model-checking/kani/pull/2561
+
+## New Contributors
+* @phayes made their first contribution in https://github.com/model-checking/kani/pull/2537
```

Comment:
This section doesn't provide a lot of value. But it'd fine with me to have this particular contribution on the "What's Changed" section.